.ci/prepare_env_utils.sh

```diff
@@ -67,7 +68,16 @@ function checkAllCreds() {
 	set +x
 }
 
-
-
-
-
+# JOB_TYPE=che-functional-tests|functional-tests|periodic-tests
```

Comment:
Now we have three jobs, so I would suggest something like ```rh-che/${JOB_NAME}``` - to have periodic and pr-check in one rh-che folder. There are e2e and bdd test so I would avoid creating many rh-che related folders in that level of hierarchy.